src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -446,6 +446,7 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			gui.doesPauseGame = _true;
 			gui.inventoryOffsetX = 20;
 			gui.inventoryOffsetY = 123;
+			gui.customBg = "";

```

Comment:
Specify a valid texture here

Reply:
If I do, we won't test without a texture. I put it empty because without that tests always fail.

Reply:
Existing GUIs will have this value null, so you need to make a way to handle null textures.

Reply:
When I tested, strangely, it didn't break my workspace.

Reply:
Maybe you set it to "" before from UI, you can't rely on this, typically it will be null

Reply:
I tested in importing a workspace created with an older version and the old GUI works fine. If this is still not for you, tell me what I should do because when I put `null`, it doesn't work (I checked everywhere in the code).